modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PhoneResourceTest.java

```diff
@@ -114,6 +114,26 @@ protected Long testGetAccountPhonesPage_getAccountId() throws Exception {
 		return _accountEntry.getAccountEntryId();
 	}
 
+	@Override
+	protected Phone
+			testGetOrganizationByExternalReferenceCodePhonesPage_addPhone(
+				String externalReferenceCode, Phone phone)
+		throws Exception {
+
+		return _addPhone(
+			phone, _organization.getModelClassName(),
```

Comment:
please use Organization.class.getName() and check if we need to update also the byId method